exercises/practice/split-second-stopwatch/.meta/src/SplitSecondStopwatch.example.elm

```diff
@@ -0,0 +1,141 @@
+module SplitSecondStopwatch exposing
+    ( State(..)
+    , Stopwatch
+    , advanceTime
+    , currentLap
+    , lap
+    , new
+    , previousLaps
+    , reset
+    , start
+    , state
+    , stop
+    , total
+    )
+
+
+type State
+    = Ready
+    | Running
+    | Stopped
+
+
+type Stopwatch
+    = Stopwatch
+        { state_ : State
```

Comment:
what are the underscores for?

Reply:
Because `state` is an exposed function, so we are not allowed to shadow it. I was a bit lazy with the `_` notation, that's the kind of thing you see in Haskell, but I didn't want to be too verbose.
Would you have a better suggestion?

Reply:
I've seen aState used before, which fits with it being an instance of the state type